frontend/components/site/siteBuildTasks.jsx

```diff
@@ -55,8 +55,19 @@ const SiteBuildTasks = () => {
   const buildId = parseInt(buildIdStr, 10);
   const [buildTasks, setBuildTasks] = useState([]);
 
+  const [build, setBuild] = useState({
```

Comment:
You can replace this useState and useEffect with the new buildDetails hook.

Reply:
Aw shoot I forgot about build tasks. On it.

Reply:
How 'bout now?